database/influxdb/influxdb_user.py

```diff
@@ -0,0 +1,220 @@
+        required: false
+    database_name:
+        description:
+            - Database name to work with.
+        required: true
+    user_name:
+        description:
+            - Name of the user that will be created/destroyed.
+        required: true
+    user_pass:
+        description:
+            - User password to create.
+        required: true
+    user_admin:
+        description:
+            - Determine if created user will be admin or not.
+        required: false
+    state:
+        description:
+            - Determines if the user should be created or destroyed.
+        choices: ['present', 'absent']
+        default: present
+        required: false
+'''
+
+EXAMPLES = '''
+# Example influxdb_user command from Ansible Playbooks
+- name: Create user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      user_name: "{{ influxdb_user_name }}"
+      user_pass: "{{ influxdb_user_pass }}"
+      user_admin: no
+      state: present
+
+- name: Destroy user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      user_name: "{{ influxdb_user_name }}"
+      state: absent
+
+- name: Create admin user
+    influxdb_user:
+      database_name: influx
+      hostname: "{{ influxdb_ip_address }}"
+      username: "{{ influxdb_username }}"
+      password: "{{ influxdb_password }}"
+      user_name: "{{ influxdb_user_name }}"
+      user_pass: "{{ influxdb_user_pass }}"
+      user_admin: yes
+      state: present
+'''
+
+RETURN = '''
+#only defaults
+'''
+
+try:
+    from influxdb import InfluxDBClient
+    from influxdb import exceptions
+    HAS_INFLUXDB = True
+except ImportError:
+    HAS_INFLUXDB = False
+
+
+def influxdb_argument_spec():
+    return dict(
+        hostname=dict(required=True, type='str'),
+        port=dict(default=8086, type='int'),
+        database_name=dict(required=True, type='str', default=False),
+        username=dict(default='root', type='str'),
+        password=dict(default='root', type='str', no_log=True),
+        user_name=dict(required=True, type='str'),
+        user_pass=dict(required=True, type='str', no_log=True),
+        user_admin=dict(required=False, default=False, type='bool')
+    )
+
+
+def connect_to_influxdb(module):
+    hostname = module.params['hostname']
+    port = module.params['port']
+    username = module.params['username']
+    password = module.params['password']
+    database_name = module.params['database_name']
+
+    client = InfluxDBClient(
+        host=hostname,
+        port=port,
+        username=username,
+        password=password,
+        database=database_name
+    )
+    return client
+
+
+def find_user(module, client, user_name):
+    u = None
+
+    try:
+        users = client.get_list_users()
+        for user in users:
+            if user['user'] == user_name:
+                u = user
+                break
+    except requests.exceptions.ConnectionError as e:
```

Comment:
`requests` is never imported. If a connection failure occurs when connecting, you'll get a NameError:

```
"msg": "NameError: global name 'requests' is not defined"
```